install_config/redeploying_certificates.adoc

```diff
@@ -437,6 +437,8 @@ by deleting existing secrets that contain service serving certificates or removi
 annotations to appropriate services.
 ====
 
+You can set the `openshift_redeploy_service_signer=false` parameter in the inventory file to skip the redeployment of service signer certificate, if required.
+If you set `openshift_redeploy_openshift_ca=true` in the inventory file, the service signing certificate is redeployed when you redeploy the master certificates.
```

Comment:
Seen from the code https://github.com/openshift/openshift-ansible/pull/12237/files#diff-f98bfc903bb7557da252f46ccb7f880098e9bd44870e135829704504b479770aR74
Only when openshift_redeploy_openshift_ca=true and openshift_redeploy_service_signer=true(default(true)) , service signing certificate would be redeployed.
If openshift_redeploy_openshift_ca=false or not set, the service signer cert is always not redeployed.

Reply:
@vwalek, will you address this comment and squash?

Reply:
@kalexand-rh  You want to change the text into `Only when openshift_redeploy_openshift_ca=true and openshift_redeploy_service_signer=true(default(true)) , service signing certificate would be redeployed. If openshift_redeploy_openshift_ca=false or not set, the service signer cert is always not redeployed.`

Reply:
@gpei, will you please take another look?